compact/optical_materials.xml

```diff
@@ -802,6 +835,31 @@
         Nlak33aMaterial->AddElement(O, natoms = 2);
       -->
     </material>
+    <material name="VM2000">
+      <D type="density" value="1.43" unit="g / cm3"/>
+      <composite n="2" ref="C"/>
+      <composite n="2" ref="H"/>
+      <composite n="2" ref="F"/>
+      <property name="RINDEX" ref="RINDEX__VM2000"/>
+      <property name="REFLECTIVITY" coldim="2"
+                values="1.776*eV  0.95
+                        4.143*eV  0.95"/>
+    </material>
+    <material name="leadtungsten_optical">
+      <D type="density" value="8.3" unit="g / cm3"/>
+      <composite n="1" ref="Pb"/>
+      <composite n="1" ref="W"/>
+      <composite n="4" ref="O"/>
+      <property name="RINDEX" coldim="2"
+                values="1.551*eV  2.4
+                        3.545*eV  2.40"/>
+      <property name="ABSLENGTH" coldim="2"
+                values="1.551*eV  200.0*cm
+                        3.545*eV  200.0*cm"/>
+      <constant name="kSCINTILLATIONYIELD1"  value="200.0/MeV"/>
+      <constant name="kSCINTILLATIONTIMECONSTANT1"    value="6.0*ns"/>
+      <constant name="RESOLUTIONSCALE"     value="1.0"/>
```

Comment:
Probably no `k` needed, and these may need to be `property`s.

Reply:
Got it. Since only one property exist in fun4all, I will ask for the properties(fast slow component of lead tungsten)

Reply:
This is why the overlap checks are failing. Probably works for your since you have an older geant4 version or so.

Reply:
Try it in `eic-shell` just to be sure.

Reply:
I change the "constant" to "property" and remove the k